Animations/include/animation/TextAnimations.h

```diff
@@ -1,13 +1,19 @@
 #ifndef MATH_ANIM_TEXT_ANIMATIONS_H
 #define MATH_ANIM_TEXT_ANIMATIONS_H
 #include "core.h"
+#include "renderer/OrthoCamera.h"
 
 struct NVGcontext;
 
 namespace MathAnim
 {
 	struct Font;
 	struct AnimObject;
+	
```

Comment:
```suggestion
	struct OrthoCamera;
```

This is the same as the comment above. Adding this should avoid the need for the include :)